src/app/tests/suites/certification/Test_TC_BI_2_1.yaml

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 18.2.1. [TC-BI-2.1] Attributes with server as DUT
+
+config:
+    cluster: "Binary Input (Basic)"
+    endpoint: 1
+
+tests:
+    - label: "Read mandatory non-global attribute: OutOfService"
+      command: "readAttribute"
+      attribute: "out of service"
```

Comment:
I Prefer all CamelCase style -> this should follow the specs and the implementation

Reply:
> If all tests are passing let go.

Attribute names sometimes don't match the spec's convention, we are using the names the YAML framework spits out when it fails to find any attribute.

Reply:
Nothing to be done currently, Test Script are passing and naming conventions are as per SDK.